flow360/plugins/report/uvf_shutter.py

```diff
@@ -572,56 +574,69 @@ class Shutter(Flow360BaseModel):
     access_token: Optional[str] = None
 
     async def _get_3d_images(self, screenshots: dict[str, Tuple]) -> dict[str, list]:
-        @backoff.on_exception(lambda: backoff.constant(3), Flow360WebNotAvailableError, max_time=3600)
-        @http_interceptor
-        async def _get_image_sequence(
-            session: aiohttp.client.ClientSession, url: str, shutter_request: list[dict]
-        ) -> str:
-            log.debug(
-                f"sending request to uvf-shutter: {url=}, {type(shutter_request)=}, {len(json.dumps(shutter_request))=}"
-            )
-            return session.post(url, json=shutter_request)
-
-        async with aiohttp.ClientSession(
-            timeout=aiohttp.ClientTimeout(total=3600),
-            headers={"Authorization": f"Bearer {self.access_token}"},
-        ) as session:
-            tasks = []
-            for _, _, shutter_request in screenshots:
-                tasks.append(
-                    _get_image_sequence(
-                        session=session,
-                        url=urljoin(self.url, "/sequence/run"),
-                        shutter_request=shutter_request,
-                    )
-                )
-                log.debug(
-                    f"request to shutter: {json.dumps(shutter_request)}"
```

Comment:
let's do not keep such legacy code in our branch. in case we need it, we could easily get them back from history. and then add the logic back.
otherwise, such code will be left there and probably never be used as dead end zone.

Reply:
you are removing code which I am using for development. How am I supposed to test report locally? It is not "legacy" code, it is still valid good which is using a different backend.

Reply:
for local test,
1.  I would suggest to config docker and run it.
2. or in you test code,  hijack _get_image_sequence function to use the different backend.
If we don't the code in prod, i would like not keep it in main branch code.

Reply:
I'm with @maciej-flexcompute we should leave it. the container still supports http so its usable code. just maybe put a comment on it explain this.

Reply:
we could add it back when we really setup the prod rest api on it. 
but for now, I would like keep the code clean. 
I will create another PR to hold such web api version in right place. 
also such docker cli code need to be moved to pipeline repo. 
most likely, I will define ShutterFetcher with WebShutterFetcher (in Flow360 repo) and CliShutterFetch(in PIpelineRepo) to keep both.

Reply:
it will get lost if we squash the history... don't we usually squash